lib/services/runtime/handlers/card.ts

```diff
@@ -63,6 +63,10 @@ export const CardHandler: HandlerFactory<BaseNode.Card.Node, typeof utilsObj> =
     }
 
     runtime.turn.set(T.CARD, newCard);
+    runtime.trace.addTrace<BaseNode.Card.TraceFrame>({
+      type: BaseNode.Utils.TraceType.CARD,
+      payload: { ...newCard, type: BaseNode.Card.CardType.SIMPLE },
+    });
```

Comment:
I think if we add the trace we can remove the `turn.set` thing, right @DecathectZero ?